tensilelite/Tensile/Tests/common/gemm/fp8_rowwise.yaml

```diff
@@ -0,0 +1,87 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx908, skip-gfx90a, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102, skip-gfx1200, skip-gfx1201] # not supported by arch
+
+GlobalParameters:
+  NumElementsToValidate: -1
+  MinimumRequiredVersion: 4.14.0
+  PrintLevel: 1
+  PrintSolutionRejectionReason: True
+  Device: 0
+  CMakeBuildType: Release
+  MergeFiles: False
+  KernelTime: True
+  MaxWorkspaceSize: 13421772800
+  DataInitTypeA: 1
```

Comment:
This test yaml is a kind of duplicated with "F8BS-TN" test of  fp8.yaml
Besides that, Do you really want to test it with DataInitTypeA = 1?

Reply:
I forgot to save the file. I'm moving it to a new file because the initialization settings are different.